notebooks/302-pytorch-quantization-aware-training/302-pytorch-quantization-aware-training.ipynb

```diff
@@ -182,7 +181,6 @@
     "    url=\"http://cs231n.stanford.edu/tiny-imagenet-200.zip\",\n",
     "    tarname=\"tiny-imagenet-200.zip\",\n",
     "):\n",
-    "    data_dir.mkdir(exist_ok=True)\n",
```

Comment:
Will it be ok if user runs it for the first time in the clean environment?

Reply:
Yes, the download_file function creates the directory if it doesn't exist. This is also tested in the CI.